src/values/callable_value.rs

```diff
@@ -104,7 +104,10 @@ impl<'ctx> AsTypeRef for CallableValue<'ctx> {
 }
 
 impl<'ctx> CallableValue<'ctx> {
+    #[llvm_versions(4.0..=13.0)]
     pub(crate) fn returns_void(&self) -> bool {
+        use llvm_sys::core::LLVMGetReturnType;
+
         let return_type =
             unsafe { LLVMGetTypeKind(LLVMGetReturnType(LLVMGetElementType(LLVMTypeOf(self.as_value_ref())))) };
```

Comment:
surely this fails on opaque pointers

Reply:
It is marked with `#[llvm_versions(4.0..=13.0)]`.